telosnetwork/teloscan
# warn about telos cloud and shedule login button to be removed

# Fixes #491

## Description
Because the Telos Cloud service will be discontinued, we must warn users with a notification and at the same time program the login button to auto-remove after the specific date (Dic 31th)

## Test scenarios
### Test the notification
- go to [this link](https://deploy-preview-492--dev-mainnet-teloscan.netlify.app/)
  - you should see a new notification type saying what's going on with Telos Cloud and a checkbox
- Dismiss without touching the checkbox
- Refresh the page
  - the notification should appear again
- Click on checkbox and then dissmiss
- reload the page
  - the notification should not appear anymore

### Test the scheduled action
- run this branch locally
```bash
git fetch --all; git checkout 491-remove-telos-cloud-wallet-by-dec-31st; yarn; yarn dev
```
- change the year from 2023 to 2022 in the file: `src/App.vue`
```typescript
export const isTodayBeforeTelosCloudDown = new Date().getTime() < new Date('2023-12-31').getTime();
```
- go to http://localhost:8080/
  - you should not see the notification
- try to connect your wallet
  - you should not see the Telos Cloud button

## Checklist:
